docs/docs/Integrations/Composio/integrations-composio.mdx

```diff
@@ -24,33 +18,38 @@ Composio components require authentication to Composio with a Composio API key.
 
 Depending on the components you use, you may also need additional access, such as an OpenAI API key, Gmail account, or GitHub account.
 
-## Use Composio components in a flow
+## Use the Composio Tools component in a flow
+
+Connect a **Composio Tools** component to an **Agent** component to enable your agent to use Gmail functionality.
 
-1. In Langflow, create a flow.
+1. In Langflow, create a Simple Agent starter flow.
 
-2. Add an **Agent** component and a **Composio Tools** component.
+2. Add a **Composio Tools** component to the flow.
 
 3. Connect the **Agent** component's **Tools** port to the **Composio Tools** component's **Tools** port.
 
-4. In the **Composio API Key** field, enter your Composio API key.
+4. In the **Agent** component, enter your OpenAI API key or configure the **Agent** component to use a different LLM.
+5. In the **Composio API Key** field, enter your Composio API key.
 
-5. In the **Tool Name** field, select the tool you want your agent to have access to.
+6. In the **Tool Name** field, select the tool you want your agent to have access to.
 
     For this example, select the **Gmail** tool to allow your agent to control an email account with the **Composio Tools** component.
```

Comment:
This is confusing to me. why would you use "Composio Tools w/ Gmail tool" vs. the "Gmail component in Tool Mode"?